dags/ft_donnees_brutes.py

```diff
@@ -0,0 +1,54 @@
+from airflow import DAG
+from airflow.decorators import task
+from airflow.operators import empty
+
+from dags.common import bucket, db, default_dag_args, slack
+
+
+with DAG(
+    "france_travail",
+    schedule_interval="@weekly",
```

Comment:
Si j'en crois le fichier `_source.yml`
> Mise à jour manuellement, via lancement d'un DAG, environ tous les 15j.

Donc plutôt :
```suggestion
    schedule_interval=None,
```